package.json

```diff
@@ -5,7 +5,9 @@
     "main": "lib/cjs/index.js",
     "typings": "lib/types/index.d.ts",
     "module": "lib/esm/index.js",
-    "sideEffects": false,
+    "sideEffects": [
+        "src/polyfills.ts"
```

Comment:
I think this needs to point to the file in the `lib` folder, since that will be bundled. So in our case it should include both the `cjs` and `esm` version.

Reply:
I wasn't sure what exactly this is pointing towards, but it makes sense to use the compiled files. Thanks